src/repository/models.py

```diff
@@ -506,6 +507,13 @@ def authors(self):
 
         return [pa.account for pa in preprint_authors]
 
+    @property
+    def safe_title(self):
+        if self.title:
+            return mark_safe(self.title)
+        else:
+            return "<Untitled>"
```

Comment:
I have a question about the use of angle brackets here: `<Untitled>`. 

Based on past experience in academia, I would expect a parenthetical like this to use square brackets like `[Untitled]` instead. That's what most of our academics would expect, to the best of my knowledge, though perhaps angle brackets would be used if you were writing in French or German or another language--I'm not sure.

I would also prefer that `<Untitled>` didn't look like an XML element in downstream data processing, where a parser would be looking for HTML or XML-like syntax. I know you haven't used `safe_title` in templates that generate XML, and Django won't think `<Untitled>` is HTML, because it's not marked safe, but is it still worth considering search crawlers, archival scrapers, and screen readers?

Let me know if angle brackets have another purpose I'm not thinking of.